packages/components/organisms/f-takeawaypay-activation/src/components/ActivationLoggedIn.vue

```diff
@@ -73,22 +73,6 @@ export default {
 
 <style lang="scss" module>
 
-.c-takeawaypayActivation-icon {
-    transform: translate(10%);
-    width: 125px;
-}
-
-.c-takeawaypayActivation-title {
-    @include font-size(body-l);
-    font-weight: $font-weight-bold;
-}
-
-.c-takeawaypayActivation-email {
-    font-weight: $font-weight-bold;
-}
-
-.c-takeawaypayActivation-actions {
-    padding-top: spacing(x3);
-}
+@import './../assets/scss/takeaway-pay-activation';
```

Comment:
Why are we moving these styles out of the component file?

Reply:
@matthewhardern They were duplicated between (sub)components and there was a suggested to extract them https://github.com/justeat/fozzie-components/pull/1078#discussion_r660828790

Reply:
makes sense